app/src/main/java/org/schabi/newpipe/util/external_communication/InternalUrlsHandler.java

```diff
@@ -49,22 +49,20 @@ public static boolean handleUrl(final Context context,
                                     final int timestampType) {
```

Comment:
After thinking more about this, I think you should split this into two functions: `handleUrlDescriptionTimestamp(Context, String)` and `handleUrlCommentsTimestamp(Context, String)`, and those functions should both call the **private** `handleUrl(Context, String, Pattern)` with the relative pattern.

Reply:
Yes, good idea.

Reply:
Done